packages/block-library/src/image/edit.js

```diff
@@ -394,6 +423,7 @@ export function ImageEdit( {
 					placeholder={ placeholder }
```

Comment:
The MediaPlaceholder has a `multiple` prop - [is that of any help here](https://github.com/WordPress/gutenberg/blob/f7e9824c2096bccbef6b498b69ec1d7bae511624/packages/block-editor/src/components/media-placeholder/index.js#L180-L180)?

Reply:
I thought about that too. Unfortunately, passing `multiple` will change the UI of the media library. I think we'll have to refactor the media library to make it not do so when only selecting one image 😅.

<img width="900" alt="image" src="https://github.com/user-attachments/assets/c4a5025b-3898-4cf7-8f38-85c5b4d4b861">

Reply:
IIRC, the media placeholder should be optimized for multiple file uploads.